src/components/checkbox/checkbox.js

```diff
@@ -167,7 +167,7 @@ function MdCheckboxDirective(inputDirective, $mdAria, $mdConstant, $mdTheming, $
         if(ngModelCtrl.$viewValue) {
           element.addClass(CHECKED_CSS);
         } else {
-          element.removeClass(CHECKED_CSS);
+          if (CHECKED_CSS) element.removeClass(CHECKED_CSS);
```

Comment:
I see, this is a constant. I just did a full search for removeClass with variables, I didn't pay attention to the details.